main.py

```diff
@@ -20,6 +20,14 @@ def main():
         line_count=2,
         column_count=16,
     )
+    # oled_display = DisplayOLED(
```

Comment:
How do you intend to allow commuting between display types? One way would be to have a `DISPLAY_TYPE` constant that is used to instantiate the correct class into a `display` variable here. Then, `display` can be passed down to `Control` in line `31`. It might require some heavier refactoring downstream though.

Reply:
I had no particular plans here and thought passing it in as configuration in code (similar to existing configuration for e.g. GPIO pins) would be fine, and I was going to leave this commented out section there as an example. Initially I was thinking of using an enum, but thought that's just hiding display-related things inside the controller, and not in fact easier than modifying `main.py`.

This is just a long-winded way of saying that I'd be happy to make changes where appropriate, but enjoyed how the controller currently doesn't need to know anything about the nature of displays and can just treat them as a black box. If we introduce an enum then we're either omitting some config (read: make the user dig into another file to change it) or need to introduce some kind of config class (which display type, what pin config) in which case we're back where we are now. Like I said, I'm very happy to make changes, I just didn't see an obviously better option.

Reply:
actually, re-reading your comment, if the question is: how would users use this, then it just works as is. So perhaps instead of labelling one as `lcd_display` and one as `oled_display` we might call both of them `display` and pass that variable in?